requirements.txt

```diff
@@ -1,110 +1,115 @@
-amqp==2.5.2
-anyjson==0.3.3
-arrow==0.7.0
-autopep8==1.2.2
-bagit==1.5.4
-celery==4.4.0
-cffi==1.11.5
-click==6.7
-coreapi==2.3.3
-coverage==4.0.3
-cycler==0.10.0
-#defusedexpat==0.4
-defusedxml==0.4.1
-Django==1.11.29
-django-appconf==1.0.2
-django-autocomplete-light==2.3.3
-django-braces==1.9.0
-django-compressor==2.2
-django-contrib-comments==1.9.0
-django-cors-headers==2.4.0
-django-crispy-forms==1.7.2
-django-debug-toolbar==1.9.1
-django-freshly==0.1.2
-django-haystack==2.8.1
-django-heartbeat==2.0.2
-django-ipware==2.1.0
-django-jsonfield==1.0.1
-django-modeltranslation==0.12.2
-django-nose==1.4.5
-django-oauth-toolkit==1.1.0
-django-robots==2.0
-django-security==0.9.3
-django-test-without-migrations==0.6
-django-timedeltafield==0.7.10
-django-widget-tweaks==1.4.1
-djangorestframework==3.11.0
-docker-py==1.7.2
-dominate==2.4.0
-DoubleMetaphone==0.1
-drf-haystack==1.8.5
-drf-yasg==1.17.1
-ecdsa==0.13.3
-enum34==1.1.6
-filebrowser-safe==0.5.0
-flake8==3.5.0
-flex==6.14.1
-flower==0.9.3
-funcsigs==1.0.2
-#functools32==3.2.3.post2
-future==0.16.0
-geojson==1.3.2
-geopy==1.16.0
-gevent==1.4.0
-grappelli-safe==0.5.1
-gunicorn==19.6.0
-html5lib==0.9999999
-inflection==0.3.1
-isodate==0.5.4
-Jinja2==2.10.1
-jsonpointer==1.14
-jsonschema==2.6.0
-keepalive==0.5
-kombu==4.6.7
-lxml==4.4.0
-#mapnik==0.1
-Markdown==3.0.1
-MarkupSafe==1.0
-matplotlib==3.1.2
-mccabe==0.6.1
-Mezzanine==4.3.1
-mock==1.3.0
-nameparser==0.5.7
-netCDF4==1.5.1.2 
-nose==1.3.7
-nose-timer>=0.7.4,<1.0.0
-oauthlib==3.1.0
-OWSLib==0.10.3
-paramiko==2.4.2
-pbr==1.10.0
-pep8==1.7.0
-prettytable==0.7.2
-probablepeople==0.5.4
-psycopg2==2.8.2
-ipaddress==1.0.22
-pyasn1==0.4.4
-pycodestyle==2.3.1
-PyCRS==1.0.1
-pyflakes==1.6.0
-pylint==1.5.5
-pyproj==1.9.5.1
-pysolr==3.4.0
-pysqlite3==0.2.1
-pytest-django>=3.4.7, <4.0.0
-python-crfsuite==0.9.6
-python-dateutil==2.5.3
-python-irodsclient==0.8.1 
-pytz==2016.3
-PyYAML==5.3
-rcssmin==1.0.6
-rdflib==4.0.1
-redis==2.10.5
-requests==2.20.0
-rfc3987==1.3.8
-robot-detection==0.4
-selenium==3.0.2
-suds-jurko==0.6
-timeago==1.0.10
-uritemplate==3.0.0
-virtualenv==15.0.2
-xmltodict==0.10.2
+# TODO: pin versions for these...
+amqp
+# anyjson
+arrow
+autopep8
+bagit
+celery
+deepdiff
+cffi
+click
+coreapi
+coverage
+cycler
+# defusedexpat support of https://github.com/tiran/defusedexpat stops at python 3.4
```

Comment:
My fork builds it for python3.6, https://github.com/sblack-usu/defusedexpat/commit/4b7fc8f90aa9523b0618152692d71cca552926a8

What python version gets installed... 3.9?  We may be able to patch the fork again.

I think this is a patch to the [xml parser](https://github.com/hydroshare/hydroshare/blob/develop/hydroshare/__init__.py#L6-L7).
```
# Import defusedexpat here so that it can patch the standard library's expat XML parser, which is used by
# rdflib and other libraries in other parts of the app.
```

Reply:
3.9
you're right, looks like your fork builds. I just read something about defusedexpat not working for > 3.4 but looks like I was in error. Thanks